src/app/modules/pages/tech/tech.component.html

```diff
@@ -7,6 +7,9 @@
   <div class="container-inner">
     <app-column [gap]="36">
       <app-column [gap]="16">
+        <em>
+          <a href="/tecnologia-en_US">Read this in English</a>
```

Comment:
Que tal a gente especificar o que será traduzido?

```suggestion
          <a href="/tecnologia-en_US">Read this page in English</a>
```

Reply:
Boa! Modifiquei aqui